draft-02.xml

```diff
@@ -969,10 +969,25 @@
             'safe', 'unsafe', or 'idempotent.'
           </t>
           <t>
-            The 'rt' attribute is OPTIONAL and is an opaque string and MAY match the
-            <xref target="prop-id" format="title"/>
-            of an
-            existing<xref target="prop-descriptor" format="title"/>.
+            The 'rt' attribute is OPTIONAL and SHOULD be one of the following:
+            <list style="numbers">
+              <t>
+                A fragment identifier (e.g. rt="#user") which points
+                to the <xref target="prop-id" format="title"/>
+                of an existing <xref target="prop-descriptor" format="title"/>
+                in the current ALPS document.
+              </t>
+              <t>
+                A resolvable URL (e.g. rt="http://example.org/profiles/user")
```

Comment:
Yeah, couldn't be and URN unless there is support for a fragment or the urn resolving to the descriptor, but can't really imagine that working. I am open to the URL, but it basically means that rt can be any old thing and a client can't rely on it to give it any semantic info reliably. It may point to some documentation or whatever, but it can't be relied on to give an unambiguous ALPS reference absent a fragment based on the current spec and fragment use.